LuaCATS/enum/ButtonPopupTypes.d.lua

```diff
@@ -1,4 +1,123 @@
 --- @meta
 
 --- @enum ButtonPopupType
-ButtonPopupTypes = {}
\ No newline at end of file
+ButtonPopupTypes = {
```

Comment:
It would be awesome if such files could be generated from C code that create that enums...